webhooks/virtualmachinewebconsolerequest/v1alpha2/validation/webconsolerequest_validator_suite_test.go

```diff
@@ -8,17 +8,17 @@ import (
 
 	. "github.com/onsi/ginkgo/v2"
 
-	"github.com/vmware-tanzu/vm-operator/pkg/lib"
+	pkgconfig "github.com/vmware-tanzu/vm-operator/pkg/config"
 	"github.com/vmware-tanzu/vm-operator/test/builder"
 	"github.com/vmware-tanzu/vm-operator/webhooks/virtualmachinewebconsolerequest/v1alpha2/validation"
 )
 
 // suite is used for unit and integration testing this webhook.
-var suite = builder.NewTestSuiteForValidatingWebhookwithFSS(
+var suite = builder.NewTestSuiteForValidatingWebhookWithContext(
+	pkgconfig.WithConfig(pkgconfig.Config{Features: pkgconfig.FeatureStates{VMOpV1Alpha2: true}}),
```

Comment:
I noticed that you used both this way and `pkgconfig.SetContext(pkgconfig.NewContextWithDefaultConfig(), func(in pkgconfig.Config)...)` to set up config across your changes. Is there any specific reason between these choices or is it just a matter of preference?

Reply:
Great question.

* `pkgconfig.NewContext` -- the config does not get defaults
* `pkgconfig.NewContextWithDefaultConfig` -- the config has defaults
* `pkgconfig.SetContext(...` -- I can provide a base config and then override a few things

Reply:
Thanks again for the detailed explanation!